### PR TITLE
Update highlightjs

### LINKED ISSUE #637: Leaf Highlight syntax is out of date
Currently built for Leaf 3, we should add support for Leaf 4

https://github.com/highlightjs/highlight.js/blob/main/src/languages/leaf.js needs updating to current spec

### PR DESCRIPTION
**Pull request by dannflor**
Updated highlightjs to 11.9.0 to include new leaf grammar. Also slimmed it down to only include grammars likely to appear in the documentation (html, css, json, etc). Close #637

### CODE REVIEW

File: docs/javascripts/highlight.min.js
Comment (gwynne) [inline docs/javascripts/highlight.min.js:1]: @dannflor I would keep the `pgsql`, `bash`, `diff`, `yaml`, and `xml` grammars also.

**Review by gwynne — CHANGES_REQUESTED** [review]

File: docs/javascripts/highlight.min.js
Comment (0xTim) [reply docs/javascripts/highlight.min.js:1]: We definitely use `diff` on the docs site

Comment (0xTim) [reply docs/javascripts/highlight.min.js:1]: To add to this, this is served by a CDN and compressed so a KB or so isn't the end of the world

Comment (dannflor) [reply docs/javascripts/highlight.min.js:1]: I went through and added all the missing grammars, there's even a `toml` block at some point

**Review by gwynne — APPROVED** [review]

**Pull request merged**